sys/net/gnrc/transport_layer/tcp/gnrc_tcp_rcvbuf.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2016 Simon Brummer <brummer.simon@googlemail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @brief       Implementation of tcp_internal/rcvbuf.h
+ *
+ * @author  Brummer Simon <brummer.simon@googlemail.com>
+ */
+#include <errno.h>
+#include "tcp_internal/rcvbuf.h"
```

Comment:
shouldn't these files rather be in `tcp/internal/` and not in `tcp/tcp_internal/`, seems a bit redundant to me 😄